### ISSUE TITLE
turn off all error message - feature request

### ISSUE THREAD

**Issue by Warter21** [zeroone2numeral2/qbittorrent-bot#57]
Hi,

I would like to turn off all error message what appears on the chat.

Thanks,
Warter

**Comment by zeroone2numeral2** [comment]
You can create a channel and redirect all exceptions notifications to that chat, by placing the channel id under `telegram.errors_log_chat` in `config.toml`

**Comment by Warter21** [comment]
Yes you are right It is good for workaround. But If it is possible I would like to rid off it completely.
